AngularJSAuthentication.Web/app/services/authService.js

```diff
@@ -1,7 +1,8 @@
 ﻿'use strict';
-app.factory('authService', ['$http', '$q', 'localStorageService', 'ngAuthSettings', function ($http, $q, localStorageService, ngAuthSettings) {
+app.factory('authService', ['$q', '$injector', 'localStorageService', 'ngAuthSettings', function ($q, $injector, localStorageService, ngAuthSettings) {
```

Comment:
Why do you prefer injecting $http using $injector instead of adding $http as a dependency?